fetch/api/request/destination/resources/dummy.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
since you are reusing this file, I guess you can just put it in the top-level /common

Reply:
I now moved the other `dummy.json` files, but I left this one here because it needs to be in the correct scope for the service worker.